CMakeLists.txt

```diff
@@ -26,54 +25,65 @@
 #
 
 project(GenomicsDB)
-cmake_minimum_required(VERSION 3.4)
+cmake_minimum_required(VERSION 3.6)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake/Modules")
 
 if (NOT CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
   message(FATAL_ERROR CMAKE_CXX_COMPILER_ID " Compiler not yet supported. Exiting.")
 endif()
 
-#External sources
+#External dependencies. Can be overridden to point to custom locations
 set(HTSLIB_SOURCE_DIR "${CMAKE_SOURCE_DIR}/dependencies/htslib" CACHE PATH "Path to htslib source directory")
 set(TILEDB_SOURCE_DIR "${CMAKE_SOURCE_DIR}/dependencies/TileDB" CACHE PATH "Path to TileDB source directory")
 set(RAPIDJSON_INCLUDE_DIR "${CMAKE_SOURCE_DIR}/dependencies/RapidJSON/include" CACHE PATH "Path to RapidJSON include directory")
 set(SPDLOG_INCLUDE_DIR  "${CMAKE_SOURCE_DIR}/dependencies/spdlog/include" CACHE PATH "Path to SpdLog include directory")
 
-#git tag
+#GIT_COMMIT_HASH derived from git tag to be used as part of GENOMICSDB_VERSION
 execute_process(
     COMMAND git log -1 --format=%h
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
     OUTPUT_VARIABLE GIT_COMMIT_HASH
     OUTPUT_STRIP_TRAILING_WHITESPACE
     )
 
-#Build parameters
-set(GENOMICSDB_RELEASE_VERSION "1.3.2-SNAPSHOT" CACHE STRING "GenomicsDB release version") #used in Maven builds
+#Build parameters/options
+set(GENOMICSDB_RELEASE_VERSION "1.3.3-SNAPSHOT" CACHE STRING "GenomicsDB release version") #used in Maven builds
 set(GENOMICSDB_VERSION "${GENOMICSDB_RELEASE_VERSION}-${GIT_COMMIT_HASH}" CACHE STRING "GenomicsDB full version string")
+
 set(DISABLE_MPI False CACHE BOOL "Disable use of any MPI compiler/libraries")
 set(DISABLE_OPENMP False CACHE BOOL "Disable OpenMP")
+
 set(BUILD_DISTRIBUTABLE_LIBRARY False CACHE BOOL "Build the TileDB/GenomicsDB library with minimal runtime dependencies")
 set(BUILD_JAVA False CACHE BOOL "Build Java/JNI interface for combined VCF records")
+
 set(HTSLIB_INSTALL_DIR "" CACHE PATH "Path to htslib install directory")
-set(OPENSSL_PREFIX_DIR "" CACHE PATH "OpenSSL installation prefix")
```

Comment:
Why delete this?

Reply:
See my comment above. I can reinstate if you want to.